server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java

```diff
@@ -1,14 +1,14 @@
-/*
+/**
```

Comment:
use /*
and seems no "<p>" marker

Reply:
I replaced /** with /*. Thanks for your suggestion.